app/src/sharedTest/java/org/oppia/android/app/home/PromotedStoryListViewModelTest.kt

```diff
@@ -0,0 +1,525 @@
+package org.oppia.android.app.home
```

Comment:
This & the other new tests should live in the same packages as the components that they're testing.

Reply:
Done